src/mappers/model.js

```diff
@@ -37,7 +37,7 @@ export const mapModelToData = (model, props) => {
     } else if (type === 'number' || type === 'slider') {
       return propValue || 0
```

Comment:
If `propValue` is equal to `0`, then you return `0` => This is not the same memory reference, but the value is `0`. 

So this is okay and **there is no bug**.

```
propValue = 0
propValue || 0
> 0
```

```
propValue = 1
propValue || 0
1
```